### PR TITLE
new|command_du: display disk usage of metrics

### PR DESCRIPTION
**Pull request by rudy-6-4**
test_command_du is coming

---
This change is [Reviewable](https://reviewable.io/reviews/criteo/biggraphite/146)

### CODE REVIEW

**Comment by rudy-6-4** [comment]
https://github.com/criteo/biggraphite/issues/92

**Comment by rudy-6-4** [comment]
Output looks like

python2 -m biggraphite.cli.bgutil du '*.*.*.*'
236.0KB carbon.agents.FR-JG5C1Y1-a.activeConnections
236.0KB carbon.agents.FR-JG5C1Y1-a.avgUpdateTime
236.0KB carbon.agents.FR-JG5C1Y1-a.blacklistMatches
236.0KB carbon.agents.FR-JG5C1Y1-a.committedPoints
236.0KB carbon.agents.FR-JG5C1Y1-a.cpuUsage
236.0KB carbon.agents.FR-JG5C1Y1-a.creates
236.0KB carbon.agents.FR-JG5C1Y1-a.droppedCreates
236.0KB carbon.agents.FR-JG5C1Y1-a.errors
236.0KB carbon.agents.FR-JG5C1Y1-a.memUsage
236.0KB carbon.agents.FR-JG5C1Y1-a.metricsReceived
236.0KB carbon.agents.FR-JG5C1Y1-a.pointsPerUpdate
236.0KB carbon.agents.FR-JG5C1Y1-a.updateOperations
236.0KB carbon.agents.FR-JG5C1Y1-a.whitelistRejects
2.0MB TOTAL

**Comment by iksaif** [comment]
Could you add a unit test ? :) (that a least make the command run, not necessarily checking its output)

File: biggraphite/cli/command_du.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Du Command."""
+
+from __future__ import print_function
+
+from biggraphite.cli import command
+
+
+_UNITS = ['B', 'KB', 'MB', 'GB', 'TB', 'PB']
```
Comment (iksaif) [inline biggraphite/cli/command_du.py]: Is there a python lib to do that ? Maybe one that we already use ? Else move it to uitls

File: biggraphite/cli/command_du.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    NAME = "du"
+    HELP = "disk usage if one or several specific metrics."
+
+    def add_arguments(self, parser):
+        """Add custom arguments.
+
+        See command.CommandBase.
+        """
+        parser.add_argument(
+            "--no-unit",
```
Comment (iksaif) [inline biggraphite/cli/command_du.py]: -h ? (like ls -h, du -h, etc..)

File: biggraphite/cli/command_du.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            help="One metric name or globbing on metrics names"
+        )
+
+    def run(self, accessor, opts):
+        """Disk usage of metrics.
+
+        See command.CommandBase.
+        """
+        accessor.connect()
+        metric_names = sorted(
```
Comment (iksaif) [inline biggraphite/cli/command_du.py]: Blocked on Dan's patch

**Pull request merged**